src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -243,6 +244,36 @@ public void WhenNewIssuesAreFound_ListenersAreUpdated()
             actualMarkers[1].Issue.RuleKey.Should().Be("S333");
         }
 
+        [TestMethod]
+        public void WhenNewIssuesAreFound_IssueFilterIsApplied()
```

Comment:
Unrelated to this PR, but I think it's a good point that we decide on UTs naming conventions:
https://dzone.com/articles/7-popular-unit-test-naming

I do MethodName_StateUnderTest_ExpectedBehavior. 
I see that you do a variation of When_StateUnderTest_Expect_ExpectedBehavior.
I also like the popular JS Should_ExpectedBehavior_When_StateUnderTest.

Which one would you prefer? I'm cool with either, just so we will be consistent.

Reply:
If I'm adding tests to an existing class (like here), I follow the naming pattern for the existing test.

I agree it would be worth standardising on naming for new test classes. I don't have a particular preference, as long as the names are not ridiculously long.